docs/Installation.md

```diff
@@ -36,6 +35,11 @@
 This document contains all steps needed to install Zonemaster::Backend. For an overview of the
 Zonemaster product, please see the [main Zonemaster Repository].
 
+If you upgrade your Zonemaster installation with a newer version of

```

Comment:
I still think there should be ha headline "Upgrade" above this text to make the reader catch that and not go to its OS directly.